languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -35467,6 +35467,26 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TOURS_CRITIQUES" name="Tours critiqués divers (barbarismes, impropriétés, solécismes, etc.)">
+      <rule id="QU_EST_CE_QUE" name="qu est-ce que" default="temp_off">
+          <antipattern>
+              <token>qu'</token>
+              <token>est</token>
+              <token>-ce</token>
+              <token regexp="yes" skip="-1">qu.*</token>
+              <token postag="M fin inte"/>
+          </antipattern>
+          <pattern>
+              <marker>
+                  <token case_sensitive="yes">qu'</token>
+                  <token>est</token>
+                  <token>-ce</token>
+                  <token regexp="yes">qu.*</token>
+              </marker>
+          </pattern>
+          <message>Dans le langage courant, cette question indirecte devrait être introduite par <suggestion>ce que</suggestion>.</message>
```

Comment:
We're going to be in trouble with the spaces here. I only see the solution to put another `<token/>` within the `<marker>` (i.e. a fifth token that can be anything) and splitting the rule.
One subrule for the case that \4 is `qu'`, in that case the suggestion is `ce \4\5`.
The second subrule for the case that \4 is `que`, in that case the suggestion is `ce \4 \5`.
Dans l'état actuel, ton premier exemple se terminerait en "... ce queil pense".

Also: if the `<marker>`s conver the entire `<pattern>`, you don't need to write `<marker>`. If you don't put any markers, the entire pattern is marked by default.